examples/polymer/vulcanize.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# vulcanize elements.html \
+#     --inline-script --inline-css --strip-comments | \
+#     crisper -h elements.build.html -j elements.build.js
+
+polybuild --maximum-crush elements/elements.html
```

Comment:
And if we actually are going to add this, please use a `package.json` script. No reason to have a file like this.